src/shared/cpu-set-util.c

```diff
@@ -82,6 +82,23 @@ char *cpu_set_to_range_string(const CPUSet *set) {
         return TAKE_PTR(str) ?: strdup("");
 }
 
+char* cpu_set_to_mask_string(const CPUSet *a) {
+        _cleanup_free_ char *str = NULL;
+        unsigned mask = 0;
+        int i, r;
```

Comment:
please add `assert(a)` here somewhere, we generally want to validate our input params superficially before we dereference them